test/spec/transmissions.spec.js

```diff
@@ -1,39 +1,116 @@
 var chai = require('chai')
   , expect = chai.expect
-  , proxyquire = require('proxyquire')
   , sinon = require('sinon')
   , sinonChai = require('sinon-chai')
-  , configuration = require('../../lib/configuration')
-  , MockRequest = require('../mocks/request.js');
+  , nock = require('nock')
+  , SparkPost = require('../../lib/index');
 
 chai.use(sinonChai);
 
 describe('Transmissions Library', function() {
-  var transmission;
+  var client, transmission;
```

Comment:
Not saying that it's a problem, but may want to be consistent with variable declaration, we do a lot of comma first stuff, so this would become

```
var client
    , transmission;
```